src/Symfony/Component/Notifier/EventListener/NotificationLoggerListener.php

```diff
@@ -51,7 +51,7 @@ public function getEvents(): NotificationEvents
     public static function getSubscribedEvents()
     {
         return [
-            MessageEvent::class => ['onNotification', -255],
+            SentMessageEvent::class => ['onNotification', -255],
```

Comment:
Why not keep both events here?

Reply:
Hey @OskarStark 
It's good to see you there :smile: 
That a good question, maybe we can keep both and add a method onNotificationSent ?

Reply:
If I had a onNotificationSent should we update `NotificationEvents` to add some methods like `addSentEvent(SentMessageEvent $event)`  and `getSentTransports()` and `getSentMessages` to store the `SentMessageEvent` ?

Reply:
After reflexion, I don't want to keep both event in NotificationLoggerListener because it add some complexity and  it's a service only used for DataCollector

Reply:
I don't change the event here, the modification of the profiler will come in another PR